cmd/orb.go

```diff
@@ -673,6 +673,17 @@ func publishOrb(opts orbOptions) error {
 		fmt.Printf("Note that your dev label `%s` can be overwritten by anyone in your organization.\n", version)
 		fmt.Printf("Your dev orb will expire in 90 days unless a new version is published on the label `%s`.\n", version)
 	}
+
+	orbResponse, err := api.OrbIsPrivate(opts.cl, namespace, orb)
+
+	if err != nil {
+		return nil
```

Comment:
Non-blocking thought: since we have an expectation that calling `api.OrbIsPrivate` with a private orb will actually return an error, what are your thoughts on wrapping this call in a function like: 
```go
func orbIsPrivate(cl *graphql.Client, namespace, orb string) bool {
    ...
    if err != nil {
        // maybe a comment explaining why an error here means
        // the orb either doesn't exist or is private
        return true
    }
    ...
}
```
This would be one way to avoid the unidiomatic `if err != nil { return nil }`.

Reply:
I wonder if we might want to make the existing api.OrbIsPrivate func return boolean instead, instead of adding more abstraction 🤔